dev/ws-proxy.ts

```diff
@@ -0,0 +1,61 @@
+import hapi from '@hapi/hapi';
+import {delay} from 'effect/Effect';
+import {CFG, E, L, pipe, RDT} from '#src/internal/pure/effect.ts';
+import {WebSocket} from 'ws';
+import {fromParameterStore} from '@effect-aws/ssm';
+import Nes from '@hapi/nes';
+
+
+const wss = await pipe(
+    CFG.redacted('/dffp/qual/dev_ws'),
+    E.provide(L.setConfigProvider(fromParameterStore())),
+    E.runPromise,
+);
+
+
+const socket = new WebSocket(RDT.value(wss));
```

Comment:
discord requires a single stable URL to forward slash command / button press inputs...

this socket connects to our API lambda and overrides further routing across backend to the local dev connection

this backdoor is only available for qual and supports only a single active connection across all devs = only 1 of us at a time can locally dev UI components.

it's possible to extend this to multiple ppl at once, maybe if we get a larger/more active dev team